dcc-submission-server/src/main/java/org/icgc/dcc/submission/server/web/controller/NextReleaseController.java

```diff
@@ -333,4 +344,17 @@ private boolean isAccessible(Authentication authentication) {
     return systemService.isEnabled() || isSuperUser(authentication);
   }
 
+  private void checkProjectsWithTransfers(List<QueuedProject> queuedProjects) {
+    val transfers = queuedProjects.stream()
```

Comment:
Ok, we can address this later then. I think having this in the service layer is most appropriate long term.